platform/android/app/src/main/kotlin/coop/polypoly/polypod/PolyStyle.kt

```diff
@@ -0,0 +1,91 @@
+package coop.polypoly.polypod
+
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+data class PolyStyle(
+    val spacing: PolySpacing = PolySpacing(),
+    val radius: PolyRadius = PolyRadius(),
+    val font: PolyFont = PolyFont(),
+    val border: PolyBorder = PolyBorder()
+)
+
+data class PolySpacing(
+    val _1x: Dp = 4.dp,
+    val _2x: Dp = (2 * _1x.value).dp,
+    val _3x: Dp = (3 * _1x.value).dp,
+    val _4x: Dp = (4 * _1x.value).dp,
+    val _5x: Dp = (5 * _1x.value).dp,
+    val _6x: Dp = (6 * _1x.value).dp,
+    val _7x: Dp = (7 * _1x.value).dp,
+    val _8x: Dp = (8 * _1x.value).dp,
+)
+
+data class PolyRadius(
+    val _1x: Dp = 4.dp,
+    val _2x: Dp = (2 * _1x.value).dp,
+    val _3x: Dp = (3 * _1x.value).dp,
+    val _4x: Dp = (4 * _1x.value).dp,
+    val _5x: Dp = (5 * _1x.value).dp,
+    val _6x: Dp = (6 * _1x.value).dp,
+)
+
+data class PolyFontFamily(
+    val jostRegular: Int = R.font.jost_regular,
+    val jostMedium: Int = R.font.jost_medium,
+)
+
+data class PolyFontSize(
+    val xs: TextUnit = 12.sp,
+    val sm: TextUnit = 14.sp,
+    val base: TextUnit = 16.sp,
+    val lg: TextUnit = 18.sp,
+    val xl: TextUnit = 20.sp,
+    val _2xl: TextUnit = 22.sp
```

Comment:
You're probably putting this underline since you can't put a number at the forefront of an identifier. But then, why not use simply xxl?
(here and in the Swift case)